index.html

```diff
@@ -5,10 +5,26 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="./css/style.css">
+  <script defer src="./js/game-scenes.js"></script>
```

Comment:
While loading in the story data like this will technically work, it's better practice to make `app.js` into a module and  import the data into it, as we demonstrated in the Bootstrap and Data code-along.